### PR TITLE
Data Lake Store File: updating the ID

### PR DESCRIPTION
**Pull request by tombuildsstuff**
Updates the ID for the Data Lake Store File so that we have all of the information required to import it. 

We also expose the `endpoint` for the Data Lake Store Account since this is useful.

This is a dependency for #1746

### CODE REVIEW

File: website/docs/r/data_lake_store_file.html.markdown [hunk 1]
```diff
@@ -43,3 +43,11 @@ The following arguments are supported:
 * `local_file_path` - (Required) The path to the local file to be added to the Data Lake Store.
 
 * `remote_file_path` - (Required) The path created for the file on the Data Lake Store.
+
+## Import
```
Comment (mbfrahry) [inline website/docs/r/data_lake_store_file.html.markdown]: It looks like this PR is missing the importer block and corresponding test

Comment (tombuildsstuff) [reply website/docs/r/data_lake_store_file.html.markdown]: yes, let me push that 🤦‍♂️

**Comment by mbfrahry** [comment]
Hitting errors 
```
--- FAIL: TestAccAzureRMDataLakeStoreFile_basic (74.39s)
	testing.go:513: Step 0 error: Error applying: 1 error(s) occurred:

		* azurerm_data_lake_store_file.test: 1 error(s) occurred:

		* azurerm_data_lake_store_file.test: Error making Read request on Azure Data Lake Store File "unlikely23exst2acctexrp.azuredatalakestore.net/test/application_gateway_test.cer" (Account "unlikely23exst2acctexrp/test/application_gateway_test.cer"): filesystem.Client#GetFileStatus: Failure sending request: StatusCode=0 -- Original Error: Get https://unlikely23exst2acctexrp/test/application_gateway_test.cer.azuredatalakestore.net/webhdfs/v1/unlikely23exst2acctexrp.azuredatalakestore.net%2Ftest%2Fapplication_gateway_test.cer?api-version=2016-11-01&op=GETFILESTATUS&tooId=true: dial tcp: lookup unlikely23exst2acctexrp: no such host
```

**Comment by tombuildsstuff** [comment]
Tests pass:

screenshot 2018-09-01 at 14 30 44

**Pull request merged**